autosubmit/platforms/paramiko_platform.py

```diff
@@ -322,16 +374,22 @@ def connect(self, as_conf: 'AutosubmitConfig', reconnect: bool = False, log_reco
                             self._ssh.connect(self._host_config['hostname'], port, username=self.user,
                                               key_filename=self._host_config_id, sock=self._proxy, timeout=60 , banner_timeout=60)
                         except Exception as e:
+                            Log.warning('SSH connect failed, will try again disabling RSA algorithms'
+                                        f'sha-256 and sha-512, error: {str(e)}')
                             self._ssh.connect(self._host_config['hostname'], port, username=self.user,
                                               key_filename=self._host_config_id, sock=self._proxy, timeout=60,
-                                              banner_timeout=60, disabled_algorithms={'pubkeys': ['rsa-sha2-256', 'rsa-sha2-512']})
+                                              banner_timeout=60,
+                                              disabled_algorithms={'pubkeys': ['rsa-sha2-256', 'rsa-sha2-512']})
                     else:
                         try:
                             self._ssh.connect(self._host_config['hostname'], port, username=self.user,
-                                              key_filename=self._host_config_id, timeout=60 , banner_timeout=60)
+                                              key_filename=self._host_config_id, timeout=60, banner_timeout=60)
                         except Exception as e:
+                            Log.warning('SSH connect failed, will try again disabling RSA algorithms'
+                                        f'sha-256 and sha-512, error: {str(e)}')
```

Comment:
Got this error in a test, and noticed it tried twice without telling the user. I guess it doesn't hurt to notify them in WARNING (or we can move to DEBUG if needed)